_test/bats-support-clone.sh

```diff
@@ -1,9 +1,10 @@
+#!/bin/bash
```

Comment:
conftest failed with this change, study how to do it

Reply:
See above. I would really ad this file to some exception list to be checked, or forget to shellscript this one all together.

Reply:
alright, remove this one from shellcheck